stylus/ui-app/forms.styl

```diff
@@ -5,9 +5,158 @@
+        width      70%
+        padding     0 15%
+        display    flex
+        margin-top 1.5em
+
+        button
+            width       100%
+            margin      0.4em
+            cursor      pointer
+
+    [role=button],
+    button
+        // mutualize behavior between button and a
+        box-sizing        border-box
+        border            0
+        display           block
+        padding           0.9em 0.75em
+        min-height        (40/basefont)em
+        border-radius     4px
+        background-color  blue
+        font-family       "Source Sans Pro", sans-serif
+        text-align        center
+        text-transform    uppercase
+        color             white
+        text-decoration   none
+
+        &.cancel
+            background-color grey-01
+            color            grey-08
+
+        &.danger
+            background-color rgba(255, 93, 70, 0.1)
+            border           solid 1px rgba(255, 93, 71, 0.1)
+            color            #ff5d47
+            margin           1em 0 3em
+            width            80%
+
+        &.flat
+            background      none
+            padding         0em
+            min-height      0
+            color           blue
+            text-transform  uppercase
+            text-decoration none
+            cursor          pointer
+
+        &[disabled]
+        &[aria-disabled=true]
+            opacity .5
+            cursor  default
+
+        &[aria-busy=true]
+            position relative
+
+            &::after
+                position     static
+                width        (16/basefont)em
+                height       (16/basefont)em
+                margin-left  (8/basefont)em
+                margin-right (-24/basefont)em
+                @extend      $icon-spinner-white
+
+
+$form-text
+    input[type=text]
+    input[type=password]
+        display       inline-block
+        padding       0.9em 0.75em 0.75em 0.9em
+        border-radius 4px
+        box-sizing    border-box
+        border        1px solid grey-03
+        background    #f5f5f5
+        color         black
+        &::placeholder
+            color     grey-07
+            font-size 1.05em
+        &:focus
+            border     1px solid blue
+            background none
+        &:hover
+            border 1px solid grey-06
+        &.error
+            border     1px solid red
+            background none
+
+
+$form-select
+    select
+        display       inline-block
+        padding       0.75em 0.75em 0.9em
+        border-radius 4px
+        box-sizing    border-box
+        border        1px solid grey-03
+        background    #f5f5f5
+        color         black
+        -moz-appearance     none
+        -webkit-appearance  none
+        appearance          none
+        background-image    url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='16px' height='16px'><polyline points='14.764,4.966 8.053,11.677 1.342,4.966'/></svg>")
```

Comment:
Isn't better to put the SVG in a dedicated file in the _assets_ directory, and reference it here with the `embedurl` function?